lib/utils/logger.js

```diff
@@ -0,0 +1,12 @@
+var _ = require('lodash');
```

Comment:
Agreed, I think that module would come out after logging spike. Let's leave this as is for time being.